api/src/util/graphqlSchema.ts

```diff
@@ -0,0 +1,152 @@
+import crypto from 'node:crypto'
+import path from 'node:path'
+
+import { getCachedDocumentNodeFromSchema } from '@graphql-codegen/plugin-helpers'
+import { CodeFileLoader } from '@graphql-tools/code-file-loader'
+import { loadSchema, LoadSchemaOptions } from '@graphql-tools/load'
+import { print, visit } from 'graphql'
+
+import { rootSchema } from '@redwoodjs/graphql-server'
+import { getConfigPath, getPaths } from '@redwoodjs/project-config'
+
+// import { logger } from 'src/lib/logger'
+
+// this seems a wrong way to import this type?
```

Comment:
Why? What's wrong?

Reply:
It's at the same level as src. For a second I was confused about api/src/types and api/types. I need api/types.